tests/src/test/java/org/mozilla/javascript/tests/NullishCoalescingOpTest.java

```diff
@@ -0,0 +1,49 @@
+package org.mozilla.javascript.tests;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.Scriptable;
+
+public class NullishCoalescingOpTest {
+
+    @Test
+    public void testNullishColascingBasic() {
+        try (Context cx = Context.enter()) {
```

Comment:
please check if you can use org.mozilla.javascript.tests.Utils#assertWithAllOptimizationLevelsES6 or similar for your test cases. At least make sure your tests are processed with all optimization levels.